debian/snapd.auto-update.service

```diff
@@ -0,0 +1,8 @@
+[Unit]
+Description=Ubuntu Core Auto Update
```

Comment:
The file names should change too. I'd probably drop the "auto-" prefix altogether here, keeping just `snapd.refresh.service` and `snapd.refresh.timer`.. the fact it's automatic is implicit from context.